src/utils/utils.ts

```diff
@@ -119,3 +119,24 @@ export function stableSort<T>(array: T[], comparator: (a: T, b: T) => number) {
   });
   return stabilizedThis.map(el => el[0]);
 }
+// @ts-ignore
+export const getEnv = forURL => {
```

Comment:
1. I don't like anything where we just use a blanket `//@ts-ignore`.
2. Rather than this, it's probably better to use the `OCL_SIGNUP_URL` constant we already have or even just evolve that to take the OCL application server name and we can generate the signup URL and this part from it.